### ISSUE TITLE
Add KTX2/Basis sample

### ISSUE THREAD

**Issue by lilleyse** [CesiumGS/3d-tiles-validator#205]
Now that KTX2/Basis support is being added to CesiumJS in https://github.com/CesiumGS/cesium/pull/9040 we should add a sample that uses KTX2/Basis textures.

@YoussefV Take a look at `createBatchedTextured` as a reference point for the new sample. Instead of using `data/wood_red.jpg` the new sample would use `data/wood_red.ktx2`. You'll mostly be working in `3d-tiles-samples-generator.ts`, `createBuilding.ts`, and `createGltf.js`.

The project is midway through changing from JavaScript to TypeScript so apologies if it's a little messy.

**Comment by YoussefV** [comment]
searching `createBatchedTextured` in this repo yields no results, I can see one usage of [`data/wood_red.jpg`](https://github.com/CesiumGS/3d-tiles-validator/blob/beaa2b51bcf7c2d87b9a4b7d7ef85c050489a10d/samples-generator/lib/createBuilding.ts#L25) here. But not sure how it relates to `createBatchedTextured`

**Comment by lilleyse** [comment]
@YoussefV not sure why your search didn't pick it up, but you can find occurrences in [`3d-tiles-samples-generator.ts`](https://github.com/CesiumGS/3d-tiles-validator/blob/master/samples-generator/bin/3d-tiles-samples-generator.ts#L140)

**Comment by lilleyse** [comment]
Yeah sorry if my message was unclear, `createBatchedTextured` is a function not a file name (in case you were doing a file search)

**Comment by YoussefV** [comment]
Yeah that's weird... Would I be replacing the `.jpg` or creating a new `[BaseColorType](https://github.com/CesiumGS/3d-tiles-validator/blob/master/samples-generator/lib/colorTypes.ts#L1)` for a `CompressedTexture`?

**Comment by lilleyse** [comment]
Just talked offline - but yes, the latter.
